Content.Shared/GameObjects/EntitySystems/SharedPullingSystem.cs

```diff
@@ -198,5 +218,25 @@ public bool IsPulling(IEntity puller)
         {
             return _pullers.ContainsKey(puller);
         }
+
+        private void UpdatePulledRotation(IEntity puller, IEntity pulled)
+        {
+            if (!pulled.TryGetComponent(out SharedRotatableComponent? rotatable))
+                return;
```

Comment:
Slap a TODO on this to update once ComponentReference works with directed event bus.